### PR TITLE
[Core] Remove skip when no numpy

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

Remove skip when no numpy

**🆕 Changelog**

- Remove skip when no numpy

### CODE REVIEW

**Comment by roigcarlo** [comment]
Numpy is now a dependency for KratosCore...

**Comment by loumalouomega** [comment]
> Numpy is now a dependency for KratosCore...

The other tests do this check, in that case we should get rid of the check on the other tests. But there is a lack of consistency. I would add the check anyway because this way if you are lacking numpy at least it run the tests instead of crashing. We can add a message of installing numpy

**Comment by loumalouomega** [comment]
> Numpy is now a dependency for KratosCore...

See:

- https://github.com/KratosMultiphysics/Kratos/blob/bcc76e7de708a8a468c2233e236a7d541dd3127d/kratos/tests/test_sparse_multiplication.py#L10
- https://github.com/KratosMultiphysics/Kratos/blob/bcc76e7de708a8a468c2233e236a7d541dd3127d/kratos/tests/test_sparse_matrices.py#L6
- https://github.com/KratosMultiphysics/Kratos/blob/bcc76e7de708a8a468c2233e236a7d541dd3127d/kratos/mpi/tests/test_distributed_sparse_matrices.py#L6
- https://github.com/KratosMultiphysics/Kratos/blob/bcc76e7de708a8a468c2233e236a7d541dd3127d/kratos/tests/test_numpy_export_dense_matrix.py#L5
- https://github.com/KratosMultiphysics/Kratos/blob/bcc76e7de708a8a468c2233e236a7d541dd3127d/kratos/tests/test_scipy_conversion_tools.py#L6

**Comment by loumalouomega** [comment]
> > Numpy is now a dependency for KratosCore...
> 
> See:
> 
>     * https://github.com/KratosMultiphysics/Kratos/blob/bcc76e7de708a8a468c2233e236a7d541dd3127d/kratos/tests/test_sparse_multiplication.py#L10
> 
>     * https://github.com/KratosMultiphysics/Kratos/blob/bcc76e7de708a8a468c2233e236a7d541dd3127d/kratos/tests/test_sparse_matrices.py#L6
> 
>     * https://github.com/KratosMultiphysics/Kratos/blob/bcc76e7de708a8a468c2233e236a7d541dd3127d/kratos/mpi/tests/test_distributed_sparse_matrices.py#L6
> 
>     * https://github.com/KratosMultiphysics/Kratos/blob/bcc76e7de708a8a468c2233e236a7d541dd3127d/kratos/tests/test_numpy_export_dense_matrix.py#L5
> 
>     * https://github.com/KratosMultiphysics/Kratos/blob/bcc76e7de708a8a468c2233e236a7d541dd3127d/kratos/tests/test_scipy_conversion_tools.py#L6

I dislike inconsistency, or we remove where it is used or we put it where it is not...

**Comment by roigcarlo** [comment]
Then we should remove the other ones and add it just as a regular import (IMHO)

**Comment by loumalouomega** [comment]
> Then we should remove the other ones and add it just as a regular import (IMHO)

In that case...

**Comment by loumalouomega** [comment]
Now @roigcarlo

File: kratos/tests/test_sparse_multiplication.py [hunk 1]
```diff
@@ -5,9 +5,11 @@
 # Additional imports
 import os
 
+import numpy as np
+
 try:
     from scipy import sparse, io
-    import numpy as np
+    import KratosMultiphysics.scipy_conversion_tools
```
Comment (loumalouomega) [inline kratos/tests/test_sparse_multiplication.py]:
```suggestion
```
Unused

**Review by philbucher — APPROVED** [review]

**Pull request merged**